<launch>
<include file="$(find livox_ros_driver2)/launch_ROS1/msg_MID360.launch"/>
<!--<include file="$(find robot_interface)/launch/start.launch"/>-->
	<include file="$(find ugas)/launch/start.launch"/>
	<include file="$(find fast_lio)/launch/mapping_mid360.launch"/>
    <include file="$(find env_sensing)/launch/start.launch"/>
    <include file="$(find nav)/launch/start.launch"/>
    <node pkg="move_base" type="move_base" respawn="false" name="move_base" output="screen" clear_params="true">
	<rosparam file="$(find nav)/config/move_base_params.yaml" command="load" />
        <rosparam file="$(find nav)/config/costmap_common_params.yaml" command="load" ns="global_costmap" />
        <rosparam file="$(find nav)/config/costmap_common_params.yaml" command="load" ns="local_costmap" />
        <rosparam file="$(find nav)/config/local_costmap_params.yaml" command="load" />
        <rosparam file="$(find nav)/config/global_costmap_params.yaml" command="load" />
        <rosparam file="$(find nav)/config/base_global_planner_params.yaml" command="load" />
        <rosparam file="$(find nav)/config/base_local_planner_params.yaml" command="load" />
        <remap from="/map" to="/static_map"/>
    </node>

</launch>
